apps/linux/tmux/tmux.talon

```diff
@@ -1,26 +1,25 @@
 os: linux
```

Comment:
Maybe also os: mac? Though once you are in a terminal you could be on any OS, might not make sense for most terminal apps to be OS bound

Reply:
I think based on the discussion here on #210, we should probably just ditch the OS entirely and let folks opt-in to tmux via the tag.